keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/api/BakeConstraint.kt

```diff
@@ -0,0 +1,5 @@
+package com.netflix.spinnaker.keel.bakery.api
+
+import com.netflix.spinnaker.keel.api.Constraint
+
+class BakeConstraint : Constraint("bake")
```

Comment:
I've renamed it to `ImageExistsConstraint` and it's now implicit. It's still required though. It's separate from the thing that launches the bake but just ensures we don't start trying to deploy an artifact before the bake has happened. We do now and the cluster resource errors because there's no AMI.

Reply:
Got it. I was under the impression the the cluster would only have a diff once the AMI existed, which is not the case.